### ISSUE TITLE
SQLAlchemy: Investigate if .ilike() should render as ILIKE

### ISSUE THREAD

**Issue by hlcianfagna** [crate/crate-python#562]
`.ilike()` is currently rendering as `lower() LIKE lower()`
Using CrateDB's native ILIKE operator could allow to benefit from a more efficient implementation in the backend in the future.

**Comment by amotl** [comment]
Good idea. The corresponding documentation of SQLAlchemy can be inspected at [^1].

> [`ColumnOperators.ilike()`](https://docs.sqlalchemy.org/en/20/core/sqlelement.html#sqlalchemy.sql.expression.ColumnOperators.ilike):
>
> Case insensitive LIKE makes use of the SQL `lower()` function on a generic backend. On the PostgreSQL backend it will use `ILIKE`.

[^1]: https://docs.sqlalchemy.org/en/20/core/operators.html#string-comparisons

**Comment by seut** [comment]
Implemented by #564.
